tinc/scripts/setup-node.sh

```diff
@@ -4,10 +4,10 @@
 set -e
 
 # Check if running as root
-if [ "$(id -u)" -ne 0 ]; then
-   echo "This script must be run as root. Try 'sudo $0'"
-   exit 1
-fi
+# if [ "$(id -u)" -ne 0 ]; then
```

Comment:
why you commented this part?
Unfortunately this script needs to setup some files that need root permissions.